java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/ui/UIContext.java

```diff
@@ -85,6 +86,9 @@ public static synchronized UIContext find() {
     protected abstract CompletableFuture<MessageActionItem> showMessageRequest(ShowMessageRequestParams msg);
     protected abstract void logMessage(MessageParams msg);
     protected abstract Message showStatusMessage(ShowStatusMessageParams msg);
+    protected CompletableFuture<String> showInputBox(ShowInputBoxParams params) {
```

Comment:
Actually, this is an API change in the API of `java.lsp.server` module. It would deserve module version increment, javadoc, `@since` tag. I am sorry for not documenting it in my commit that introduced this method.